docs/mining/overview.md

```diff
@@ -26,74 +26,141 @@ transactions waiting to be processed in the mempool.
 
-These mining pools are known to support Decred:
+Over the past few years many hosting companies have become available in North America and can be found in this [database](https://hashrateindex.com/farms). This database contains some of the top facilities like Blockware, Compute North, Core Scientific, Frontier, Box Miner, and many more. 
 
-* [:fontawesome-solid-external-link-square-alt: BeePool](https://beepool.org)
-* [:fontawesome-solid-external-link-square-alt: F2Pool](https://www.f2pool.com)
-* [:fontawesome-solid-external-link-square-alt: Poolin](https://www.poolin.com)
-* [:fontawesome-solid-external-link-square-alt: UUpool](https://uupool.cn/dcr)
-* [:fontawesome-solid-external-link-square-alt: Luxor](https://mining.luxor.tech/coins/decred)
-* [:fontawesome-solid-external-link-square-alt: Coinmine](https://www2.coinmine.pl/dcr/)
-* [:fontawesome-solid-external-link-square-alt: Suprnova](https://dcr.suprnova.cc)
+Please make sure to do your own research before making any decision on engaging a colocation business.
 
-Mining pools all work more or less the same but you may wish to sign up at multiple pools and see which one suits you the best.
 
-Please choose to mine on a smaller pool so the network's hashrate can be spread out for better decentralization of mining power!
+## Setting up your ASIC
 
----
+**Powering up the Miner**
+
+Connect your machine to the power outlet via the built-in PSU. Then connect your miner to your internet-connected router or switch using a standard network cable. You are now ready to power up your miner.
 
-## <img class="dcr-icon" src="/img/dcr-icons/Servers.svg" /> GPU Drivers/Software
+**Scanning for Devices**
 
-GPU drivers usually contain the libraries needed for mining.  If you have difficulties running the software you may wish to re-install and specifically check that the OpenCL (AMD) or CUDA (NVIDIA) libraries are selected.
+To get started, use a scanning tool like [AngryIP](https://angryip.org/) or [Locator](https://minerstat.com/software/locator) to scan every device on your local network and see its IP address. After you have identified your miner’s IP address, you are ready to proceed with Configuration.
+
+**Configuration**
+
+Enter the local network IP address of your miner in the URL bar of any web browser. A dashboard similar to the one shown below will pop up. We are setting up a Whatsminer D1, but the process is similar for other ASICs.
+
+
+![Rig Configuration Detail](../img/pow-mining-guide/rig_config.png?raw=true)
 
 ---
 
-## <img class="dcr-icon" src="/img/dcr-icons/Download.svg" /> Select and Download Mining Software
+## Solo Mining or Pool Mining
```

Comment:
Remove this whole section IMO - everything under both the "Solo Mining" and "Pool Mining" headers can go.

Reply:
Removed

Reply:
The section still seems to be there, but thats fine.